src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+h1 {
+  display: none;
+}
+
+.calendar {
+  margin: 40px auto 0;
+  width: ($cell-size + 1) * $coll-couny;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__cell {
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+
+    text-align: center;
+    line-height: $cell-size;
+    font-size: 30px;
+
+    border: 1px solid black;
+    background-color: #eee;
+
+    transition-duration: 500ms;
```

Comment:
Add the property you want to transition, like 'transition: transform 500ms;' for better performance.